packages/amplify-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -33,8 +34,16 @@ export const amplifyRegions = [
 
 export function initJSProjectWithProfile(cwd: string, settings: Object) {
   const s = { ...defaultSettings, ...settings };
+  let env;
+
+  if (s.disableAmplifyAppCreation === true) {
```

Comment:
Are these circleCI settings?

Reply:
No, this is the option tests use to control wether the env var will be set for launched CLI or not.